packages/core/src/components/HelpDrawer/HelpDrawerToggle.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * A link that triggers the visibility of a help drawer
+ */
+export class HelpDrawerToggle extends React.PureComponent {
+  render() {
+    if (!this.props.helpDrawerOpen && this.buttonRef) {
+      this.buttonRef.focus();
+    }
+    const blockInlineClass = `ds-u-display--${
+      this.props.inline ? 'inline-block' : 'block'
+    }`;
+    /* eslint-disable jsx-a11y/anchor-is-valid */
+    return (
+      // Use a <span> since a <div> may be invalid depending where this link is nested
+      <span className={blockInlineClass}>
+        <a
+          href="javascript:void(0);"
+          className={this.props.className}
+          ref={el => (this.buttonRef = el)}
+          onClick={() => this.props.showDrawer()}
+        >
+          {this.props.children}
+        </a>
+      </span>
+    );
+  }
+}
+
+/* eslint-disable react/no-unused-prop-types */
+HelpDrawerToggle.propTypes = {
+  /** Whether or not the Help Drawer controlled by this toggle is open or closed. This value is used to re-focus the toggle that opened the drawer when the drawer closes. */
```

Comment:
What's the intended behavior with regards to focusing? When you open the help drawer, is it supposed to move focus back to the button rather than focus on the help drawer's title?

Reply:
It should focus the title of the help drawer on open (I'm not entirely sure why, this was behavior that I duplicated from the original - @1Copenut can give a better answer, but maybe because the title is read out by screen readers and it makes more sense for it to first read the title rather than immediately reading the close button?) and on close it should return focus to the toggle that opened it (so you're back to your original spot in the DOM before it opened.)

Reply:
@hannaliebl @pwolfert 
You're exactly right. Settling focus on the heading when the Help Drawer is opened causes it to be read out immediately to the big three screen readers. NVDA doesn't always read out content when we focus a container or non-text node, so I'm trying to rally around heading focus for programmatic UI changes where possible, and fall back to non-text nodes where necessary.

Here we had a chance to hook into the heading, and it's maximally useful to all assistive devices I tested against.